### PR TITLE
Make it a bit easier to run the tests

### PR DESCRIPTION
**Pull request by jaseemabid**
Improve docs, mention insecure registries and use the same command everywhere.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -152,22 +152,23 @@ This snippet will ensure your new service will skip deployment to production, bu
...
 
-There are couple tests for SaaS Herder. To run them, you can use included `tests/Dockerfile.test`
+SaaS Herder tests are run in an isolated container, but talks to the host docker
+daemon to talk to the oc cluster.
 
-```
-docker build -t saasherder-test -f tests/Dockerfile.test .
-```
+NOTE: You might have to additionally enable
+[Insecure Registries](https://wiki.archlinux.org/index.php/Docker#Insecure_registries) on
```
Comment (vpavlin) [inline README.md]: Mind adding a not that the link is only for Arch Linux? (Fedora and Centos can be easily configured through sysconfig)

**Comment by vpavlin** [comment]
[test]

**Comment by vpavlin** [comment]
[test]

**Comment by vpavlin** [comment]
[test]

**Comment by kbsingh** [comment]
@vpavlin  are the tests run on a duffy node or on the slave itself ? if its the slave, we might need to think through it a bit since there is no docker support there.

**Comment by vpavlin** [comment]
[test]

**Comment by vpavlin** [comment]
They run on duffy

**Pull request merged**